app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsController.kt

```diff
@@ -74,14 +74,8 @@ class PatientSearchResultsController @Inject constructor(
   }
 
   private fun saveEntryAndGoToRegisterPatientScreen(ongoingNewPatientEntry: OngoingNewPatientEntry, currentFacility: Facility): Observable<UiChange> {
-    val nextScreenChange = if (isFacilitySwitchedPreference.get()) {
```

Comment:
The `isFacilitySwitchedPreference` dependency is now unused.

Reply:
Good catch!